content/docs/introduction/glossary.md

```diff
@@ -102,9 +106,9 @@ A remote write endpoint is what Prometheus talks to when doing a remote write.
 
 ### Silence
 
-A silence in the Alertmanager prevents alerts with labels matching the silence from
+A silence in the Alertmanager prevents alerts, with labels matching the silence, from
 being included in notifications.
 
 ### Target
 
-One application, server or endpoint that Prometheus is scraping.
+The method by which Prometheus scrapes an instance.
```

Comment:
HTTP is the method by which we scrape. A target is how we scrape an endpoint, things like the labels it has, auth etc.

Reply:
Ah now I get it! I'll rewrite based on your comments. Thanks - it was very obtuse. :)